packages/ui/v2/core/Shell.tsx

```diff
@@ -519,14 +523,20 @@ const MobileNavigation = () => {
     <>
       <nav
         className={classNames(
-          "bottom-nav fixed bottom-0 z-30 -mx-4 flex w-full bg-white shadow md:hidden",
+          "bottom-nav fixed bottom-0 z-30 -mx-4 flex w-full border border-t border-gray-200 bg-gray-50 shadow md:hidden",
           isEmbed && "hidden"
         )}>
         {navigation
           .filter((i) => i.href !== "/settings/profile")
-          .map((item, itemIdx) => (
-            <MobileNavigationItem key={item.name} item={item} itemIdx={itemIdx} />
-          ))}
+          .map((item) => !item.more && <MobileNavigationItem key={item.name} item={item} />)}
+
+        <MobileNavigationItem
+          item={{
+            name: "more",
+            href: "/more",
+            icon: Icon.FiMoreHorizontal,
+          }}
+        />
```

Comment:
this was my best guess at adding a "more" icon without adding it to the desktop navigation. not sure if this is the best approach but it works

Reply:
Will make some adjustments

Reply:
in this PR or followup?

Reply:
This one. You should be sleeping mate

Reply:
Made some adjustments, routing forms shouldn't display if not installed 
<img width="422" alt="image" src="https://user-images.githubusercontent.com/3504472/188759090-fa2991e0-7bbb-45a4-903b-c6c6ebd3c2f1.png">